content/docs/accessibility.md

```diff
@@ -1,32 +1,33 @@
 ---
 id: accessibility
-title: Accessibility
+title: נגישות
 permalink: docs/accessibility.html
 ---
 
-## Why Accessibility? {#why-accessibility}
+## למה נגישות? {#why-accessibility}
 
-Web accessibility (also referred to as [**a11y**](https://en.wiktionary.org/wiki/a11y)) is the design and creation of websites that can be used by everyone. Accessibility support is necessary to allow assistive technology to interpret web pages.
+נגישות ברשת (ידועה גם כ [**a11y**](https://en.wiktionary.org/wiki/a11y)) היא יצירה ועיצוב אתרים שמתאימים לשימוש ע״י כולם. תמיכה בנגישות נדרשת ע״י טכנולוגית מסייעת כדי לפרש דפי אינטרנט.
 
-React fully supports building accessible websites, often by using standard HTML techniques.
+React מספקת תמיכה מלאה בבניית אתרים נגישים, בדרך כלל ע״י שימוש בטכניקות HTML סטנדרטיות.
 
-## Standards and Guidelines {#standards-and-guidelines}
+## סטנדרטים וקווים מנחים {#standards-and-guidelines}
 
 ### WCAG {#wcag}
 
-The [Web Content Accessibility Guidelines](https://www.w3.org/WAI/intro/wcag) provides guidelines for creating accessible web sites.
+ניתן למצוא קוים מנחים ליצירת אתרים נגישים ב[Web Content Accessibility Guidelines](https://www.w3.org/WAI/intro/wcag).
 
-The following WCAG checklists provide an overview:
+הרשימה הבאה מ WCAG מספקת סקירה כללית:
 
 - [WCAG checklist from Wuhcag](https://www.wuhcag.com/wcag-checklist/)
```

Comment:
I think these 3 bullets should be translated aswell, something like:
Wuhcag -מ WCAG רשימת